cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/parameters/ParameterUtils.java

```diff
@@ -297,14 +297,22 @@ public static boolean hasValidParameters(HttpServletRequest request, HttpServlet
     return true;
   }
 
+  /**
+   * Returns the case sensitive request parameter name, or <code>null</code> if the parameter does not exist.
+   */
+  private static String caseSensitiveParameterName(HttpServletRequest request, String parameter) {
```

Comment:
should the function be named caseInsensitiveParameterName()? Since we return case insensitive param name. Also, how do we 'not' specify a parameter here?

Reply:
@alexwang08 
* I believe we return the case sensitive name here -- do you think there is a bug in the logic?
* I am not sure `how do we 'not' specify a parameter here` means -- do you mean what happens if the parameter name does not exist in the request? If so, it is documented in the JavaDoc